apps/common_logging_processors.go

```diff
@@ -88,3 +88,63 @@ func genericAccessLogParser(ctx context.Context, processorType, tag, uid string)
 	c = append(c, mf.Components(ctx, tag, uid)...)
 	return c
 }
+
```

Comment:
Please add a comment describing the difference between this new function and a `TODO:` above `genericAccessLogParser` to "remove" when all references to it have been updated.